consensus/polybft/contracts_initializer.go

```diff
@@ -76,6 +76,41 @@ func getInitChildERC20PredicateInput(config *BridgeConfig) ([]byte, error) {
 	return params.EncodeAbi()
 }
 
+// getInitChildERC20PredicateAccessListInput builds input parameters for ChildERC20PredicateAccessList SC initialization
+func getInitChildERC20PredicateAccessListInput(config PolyBFTConfig) ([]byte, error) {
+	//nolint:godox
+	// to be fixed with EVM-541
+	// TODO: @Stefan-Ethernal Temporary workaround just to be able to run cluster in non-bridge mode, until SC is fixed
+	rootERC20PredicateAddr := types.StringToAddress(disabledBridgeRootPredicateAddr)
+	rootERC20Addr := types.ZeroAddress
+
+	if config.Bridge != nil {
```

Comment:
I think you can remove this condition an TODO when you merge with `develop` since RFC-201/205 changes are merged. We will always have `Bridge` configured.

Reply:
Yes, but it means we can remove the previous 2 lines as well and that's out of the scope of the PR, adding TODO